pom.xml

```diff
@@ -56,6 +56,11 @@
             <groupId>org.projectlombok</groupId>
             <artifactId>lombok</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.liquibase</groupId>
+            <artifactId>liquibase-core</artifactId>
+            <version>${liquibase.version}</version>
```

Comment:
First, you don't create alias liquibase.version in properties your pom file.
Second, spring-boot-starter-parent already include liquibase, so you don't need to specify the version here.